trr.el

```diff
@@ -230,10 +230,10 @@ and be writable by `update-game-score' helper program. "
     (setq num (+ (/ num1 2) (/ num2 2)))
     (% num TRR:number-of-text-lines)))
 
-
-;; files used by TRR
-(defvar TRR:update-program (expand-file-name "update-game-score" exec-directory))
-(defvar TRR:select-text-file (expand-file-name "CONTENTS" TRR:directory))
```

Comment:
`(defvar TRR:select-text-file (expand-file-name (if TRR:japanese "CONTENTS.ja" "CONTENTS") TRR:directory))`
does not work. `TRR:japanese` in user emacs config is not yet loaded.